packages/govuk-frontend/src/govuk/components/header/_index.scss

```diff
@@ -34,10 +34,14 @@
 
   .govuk-header__logotype {
     display: inline-block;
+    position: relative;
+    top: -2px;
```

Comment:
Great to see this in draft 🙌

Just a reminder we'll need `-3px` for Chrome and macOS Safari as it's 1px too low here as well

But based on https://github.com/alphagov/govuk-frontend/pull/4423 we could use `-2.5px` via Firefox CSS hack to snap to pixels

Reply:
Does the same apply to this version of the lockup? 

Part of why it's still in draft is because I was gonna see if it had the same problems (though tomorrow, given the time).

Reply:
Yeah unfortunately, have a look at the [Percy diff](https://percy.io/55ba5e1a/govuk-frontend/builds/31103048/changed/1714881958?browser=edge&browser_ids=49%2C50&subcategories=unreviewed%2Cchanges_requested&viewLayout=overlay&viewMode=new&width=1280&widths=375%2C1280) 😔

With this crown SVG it's even stranger:

Firefox shows the crown (but not GOV.UK) shifts 1px lower versus baseline
Edge shows the crown and GOV.UK shifts 1px shift versus baseline

Reply:
I'm not _too_ concerned with the text baseline shifting between old/new versions, so long as the new is still visually aligned with the product name when the two appear together. At least on my visual test, the current PR and #4297 in their current state render them as desired in Chromium, Firefox and iOS Safari. It is misaligned by one pixel in macOS Safari. 

From the sounds of it, subpixel rounding is still playing a part, otherwise I don't know why the crown would have shifted in Firefox but not the text. 

Given both this and the Tudor crown PR have already been tweaked to snap many elements to whole coordinatates, and the SVG's artboard and viewbox exactly align to the on-screen dimensions, I'm not sure there's anything much more we could do to negate the possibility of subpixel rounding. Ultimately it's just a consequence of user agents rendering graphics differently to how they render regular text. 

This then opens the floor to a persistent effort to make micro-adjustments for different browsers, operating systems, screen resolutions and densities, zoom levels, et al., which may need to keep being adjusted as browsers update, and I'm not sure we would want to open that can of worms.

I'm sort of agreeing with [Owen's comment last week](https://github.com/alphagov/govuk-frontend/pull/4297#issuecomment-1792304864) that this just might not be a big enough problem to an end user (if it's even noticeable) to really invest too much time in resolving; or at least it's probably something that can wait for later.

Reply:
Yeah, true. See what design feedback you get

Don't forget there's still `display: inline` applied too soon (fixed in https://github.com/alphagov/govuk-frontend/pull/4423/commits/afb353228c648c8420d3b539075688b7abc12dfd) and a missing `vertical-align` on `.govuk-header__product-name` (fixed in https://github.com/alphagov/govuk-frontend/pull/4423/commits/f1d373ef11fcefd9d67a2fea88b231004468b34b) which is fighting with the current approach

I'm still happy with a quick Firefox CSS hack ([via my spike](https://github.com/alphagov/govuk-frontend/pull/4423)) to bring everything into alignment

Should the Firefox CSS hack stop working one day, we're back to what you described

Reply:
So, the questions I would ask here are:

1. Is this a massive hassle to solve?
2. What is the impact to users?

For 1, I don't see harm in exploring this to see if there's something more robust we can do. It sounds like we've tried to explore this a little bit but it's becoming maybe too complex to be worth it. For 2, my take as someone who just has opinions and isn't designer is that whilst it's a disturbance, it'd assert that it wouldn't bring a user out of their journey in any fashion.

A 3rd question I'd ask is: what is the user case for switching to different browsers that would encourage any degree of scrutiny from a user? My point being that it's ok for things to look different in different browsers. I understand the perspective of wanting to line up the thing but I think if we do our due diligence to make this as close as possible, specifically the difference between browsers isn't something worth stressing about IMO.

I'm open to hearing compelling evidence to the contrary. I reckon the way forward is to see if there's something clever we can do to make this less problematic, jot up a comment somewhere explaining that this is a known problem.